pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -720,7 +720,7 @@ def get_chemical_indices(self):
             numpy.ndarray: A list of chemical indices
 
         """
-        return self.indices
+        return self.indices.copy()
```

Comment:
Is this related to this pull request?

Reply:
This is from https://github.com/pyiron/pyiron_atomistics/pull/350